deployment/configCmd.py

```diff
@@ -176,13 +176,15 @@ def push_configuration(self, args):
         if args.kube_config_path != None:
             args.kube_config_path = os.path.expanduser(args.kube_config_path)
         push_list = [
+            "config.yaml",
```

Comment:
If `config.yaml` doesn't exist, what will happen?

Reply:
The same as `layou.yaml`, the logger will raise an IOError: `[Errno 2] No such file or directory: './tmp-config-1616644286/layout.yaml'`

Reply:
This will a breaking change. 

Please add the following text if `config.yaml` doesn't exist:


Cannot find config.yaml in your config folder. Before v1.7.0, this file is stored in ~/pai-deploy/cluster-cfg/config.yaml on the dev box machine. 
If you are upgrading from v1.7.0, please copy this file to the config folder.